### PR TITLE
use a real parser to check validity

### PR DESCRIPTION
**Pull request by sprsquish**

### CODE REVIEW

File: tests/fauna/DeployQueries.test.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ describe('Fauna deploy', () => {
   let dbRef
   const BaseFQLValue = new values.Query({
     lambda: 'ref',
-    expr: [ { var: 'ref' }, "this/is/not/a/comment" ],
+    expr: [{ var: 'ref' }, "this/is/not/a/comment", "//this is also not a comment"],
```
Comment (cleve-fauna) [inline tests/fauna/DeployQueries.test.js:31]:
```suggestion
    expr: [{ var: 'ref' }, "this/is/not/a/comment", "//this is also not a comment", "/* also not a comment */"],
```

File: tests/test.data.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
 const { query: q } = require('faunadb')
 
-const BaseFQL = q.Lambda('ref', [ q.Var('ref'), "this/is/not/a/comment" ])
+const BaseFQL = q.Lambda('ref', [
+  q.Var('ref'),
+  "this/is/not/a/comment",
+  "//this is also not a comment"
+])
```
Comment (cleve-fauna) [inline tests/test.data.js:7]:
```suggestion
  "//this is also not a comment",
  "/* also not a comment */"
])
```

File: tests/test.data.js [hunk 3]
```diff
@@ -10,7 +15,8 @@ Lambda(
   // Comment
   [
     Var("ref" /* Inline comment */),
-    "this/is/not/a/comment"
+    "this/is/not/a/comment",
+    "//this is also not a comment"
   ]
```
Comment (cleve-fauna) [inline tests/test.data.js:20]:
```suggestion
    "//this is also not a comment",
    "/* also not a comment */"
  ]
```

**Comment by pnwpedro** [comment]
- [ ]

**Pull request closed without merging**